modules/VersionInfo.cmake

```diff
@@ -21,7 +21,11 @@ else()
 	set( BUILD_VER 0 )
 endif()
 if(NOT BUILD_DATE)
-	string(TIMESTAMP BUILD_DATE "%d.%m.%Y")
+	if (DEFINED ENV{SOURCE_DATE_EPOCH})
+		execute_process(COMMAND "date" "-u" "-d" "@$ENV{SOURCE_DATE_EPOCH}" "+%d.%m.%Y" OUTPUT_VARIABLE BUILD_DATE OUTPUT_STRIP_TRAILING_WHITESPACE)
```

Comment:
Shouldn't this be scoped to ... non-windows platforms only?

Reply:
Sorry for late answer, CMake claims that SOURCE_DATE_EPOCH variable is already used
https://cmake.org/cmake/help/latest/command/string.html
Note If the SOURCE_DATE_EPOCH environment variable is set, its value will be used instead of the current time. See https://reproducible-builds.org/specs/source-date-epoch/ for details.